### DISCUSSION TITLE
I need help with generating the values for ENV file

### DISCUSSION THREAD

**Discussion by hammoudi00** [Shikhar15606/ElectionDapp#143] [Q&A] [answered]
Can anyone help me please how to populate values for env file I'm not able to figure out from where to get the values that needs to be provided.

I'm still newbie to blockchain development and any help will be much appreciated.

**Comment by Shikhar15606** [comment, marked as answer]
Hi @hammoudi00 :wave: 
Here is the complete setup guide for this project [https://trapezoidal-calendula-968.notion.site/Setting-up-the-Project-0278e33f0c224ae4aea97b421e3240f3](https://trapezoidal-calendula-968.notion.site/Setting-up-the-Project-0278e33f0c224ae4aea97b421e3240f3)

Hope it helps :wink:

**Reply by hammoudi00** [reply]
Thank you in advance brother 
i have problem with Twilio  after register they suspend my account ???!!!
